.github/workflows/packages_tests.yaml

```diff
@@ -48,6 +48,9 @@ jobs:
             -   run: composer config minimum-stability dev
 
             # test with current commit in a pull-request
+
+            -   run: composer require --dev rector/rector-generator:dev-main
```

Comment:
after https://github.com/sabbelasichon/typo3-rector/pull/2755 tweak, this tweak will no longer needed.